### ISSUE TITLE
Allow users to save downloads to multiple folders

### ISSUE THREAD

**Issue by MetalNeo** [ccrama/Slide#3403]
Slide version: 6.7.1
Android version: 11

I like how Slide allows you to choose your own folder to save downloads, along with the option for subreddit folders. However, I think it would be convenient if users had the option of adding additional folders to download to with the idea of giving users better categorization/management of their downloaded images.

For example, let's say you want to store memes in a folder for memes. But, you want to store artwork in another folder to keep them more organized.

I could see this being implemented where the user can add additional download folders/locations in settings. Then when users download an image/video, if there is more than one download folder in the settings, then a popup window could appear with a list of the folders. The user would then select which folder they would like to save to, and the file would be saved to that folder.

**Comment by TacoTheDank** [comment]
I don't think this is functionally possible, unless you mean something different then what I'm thinking you mean. Perhaps a better way would be to add a setting that, when switched on, allows you to choose the download location each time the download button is pressed.

**Comment by MetalNeo** [comment]
Yeah, that would work too (and probably be a simpler solution). The key thing is being able to save images in different folders with relative ease.
